src/modules/components/member/memberAvatar/memberAvatar.tsx

```diff
@@ -0,0 +1,63 @@
+import * as blockies from 'blockies-ts';
+import type React from 'react';
+import { getAddress, isAddress, type Hash } from 'viem';
+import { normalize } from 'viem/ens';
+import { useEnsAddress, useEnsAvatar, useEnsName } from 'wagmi';
+import { Avatar, type IAvatarProps } from '../../../../core';
+
+export interface IMemberAvatarProps extends Omit<IAvatarProps, 'fallback'> {
+    /**
+     * ENS name of the user to lookup avatar src.
+     */
+    ensName?: string;
+    /**
+     * 0x address of the user to look up ENS name and avatar src.
+     */
+    address?: string;
+    /**
+     * Direct URL src of the user avatar image to be rendered.
+     */
+    avatarSrc?: string;
+}
+
+export const MemberAvatar: React.FC<IMemberAvatarProps> = (props) => {
+    const { ensName, address, avatarSrc, size = 'sm', className, ...otherProps } = props;
```

Comment:
We can just pass these attributes as `otherProps`
```suggestion
    const { ensName, address, avatarSrc, ...otherProps } = props;
```